Fix The "<" not displayed for some blockchains [IOS-2821]

skibinalexander opened the pull request:
Добавить знак < в поле комиссии для

- Для блокчейна ETH POW
- Для токенов ETH POW
- Только для токенов Avalanche

skibinalexander committed:
Fix The "<" not displayed for some blockchains

Добавить знак < в поле комиссии для

- Для блокчейна ETH POW
- Для токенов ETH POW
- Только для токенов Avalanche

tureck1y commented on BlockchainSdk/Common/Blockchain.swift:
```diff
@@ -247,6 +247,14 @@ public enum Blockchain: Equatable, Hashable {
             if case .token = amountType {
                 return true
             }
+        case .ethereumPoW:
```
для ethereumPoW должно быть всегда true, можно объединить с первым кейсом, а аваланч со вторым)

skibinalexander replied:
Ага, понял. В задаче чутка странно было написано;

skibinalexander committed:
Поправил замечания к ПР

для ethereumPoW должно быть всегда true, можно объединить с первым кейсом, а аваланч со вторым)
